pyccel/complexity/arithmetic.py

```diff
@@ -63,7 +62,7 @@ def count_ops(expr, visual=None):
         a = expr.iterable.size
         ops = sum(count_ops(i, visual) for i in expr.body.body)
         return a*ops
-    elif isinstance(expr, (Tuple,PythonTuple)):
+    elif isinstance(expr, Tuple):
```

Comment:
Do we still need to use `Tuple` here?

Reply:
The files in this folder begin with a call to pyccel_to_sympy

Reply:
I see, so we need to use Sympy in certain places in order to perform specific operations. This should be clarified in the PR's title and description. Maybe change the title to "Remove sympy.Tuple from Pyccel's AST"?

Reply:
I don't think this file is used by pyccel at the moment. Sympy should be kept where it is used as designed, ie to simplify equations etc

Reply:
Fine with me.